hudi-cli/src/main/scala/org/apache/hudi/cli/DedupeSparkJob.scala

```diff
@@ -152,7 +209,7 @@ class DedupeSparkJob(basePath: String,
       val newFilePath = new Path(s"$repairOutputPath/${fileNameToPathMap(fileName).getName}")
       LOG.info(" Skipping and writing new file for : " + fileName)
       SparkHelpers.skipKeysAndWriteNewFile(instantTime, fs, badFilePath, newFilePath, dupeFixPlan(fileName))
-      fs.delete(badFilePath, false)
+      fs.delete(badFilePath, true)
```

Comment:
Can you explain this change?

Reply:
We need to set it to true because somehow xyz.parquet.bad is getting considered as a directory and not a file. Hence to be able to delete, we need to set recursive to true. @yanghua